addons/settings/fnc_initDisplayMain.sqf

```diff
@@ -29,8 +28,13 @@ if (_file != "") then {
     };
 
     if (_fileExists) then {
+        INFO_1("Userconfig: Loading file [%1]",_file);
         _userconfig = preprocessFile _file;
+    } else {
+        INFO_1("Userconfig: Could not open file [%1]",_file);
```

Comment:
Works, but technically the file could also be just empty. Probably not worth mentioning it.

Reply:
`Could not open file [%1] (empty or <other reason>)`?

Reply:
File not found or empty.
This also reminds me that the log comments should have periods at the end imo.